src/presentation/components/team/Member.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <li class="member">
+    <div class="member-name">
+      {{ teamMember.user.name }}
```

Comment:
what if user does not have a name?

Reply:
As I know name it is field what cannot be empty

Reply:
But if this case may happend I can add someting like "user + user number" instead his name
Or create field nickname in the User entity

Reply:
What case do you think is the best?

Reply:
Maybe use email? 
It is required for authentication